js/src/primary-menu.es6.js

```diff
@@ -8,7 +8,7 @@ import Drupal from 'drupal';
 
   Drupal.behaviors.primaryNav = {
     attach(context) {
-      const submenuNode = context.querySelector('.menu--main');
+      const submenuNode = context.querySelector('.menu--submenu');
```

Comment:
I don't think this is correct. The MenuBar should be the horizontal menu with the top-level links, which is menu--main. (Why the variable is called submenuNode is an excellent question. I think we should change that to something more sensible like menuNode.) I'm basing this off of https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html, where the top-level menu has the role of 'menubar'.

Reply:
That makes sense. I'll update.